Patches/Patches_VanillaFactionExpandedVikings.xml

```diff
@@ -8,25 +8,112 @@
     <match Class="PatchOperationSequence">
       <success>Always</success>
       <operations>
+        <!-- Function Extension-->
+        <li Class="PatchOperationAdd">
+          <xpath>Defs/ThingDef[defName="VFEV_FurBed" or defName="VFEV_DoubleFurBed"]</xpath>
+          <value>
+            <inspectorTabs>
+              <li>SoftWarmBeds.ITab_Bedding</li>
+            </inspectorTabs>
+          </value>
+        </li>
+        <li Class="PatchOperationAdd">
+          <xpath>Defs/ThingDef[defName="VFEV_FurBed" or defName="VFEV_DoubleFurBed"]/building</xpath>
+          <value>
+            <fixedStorageSettings>
+              <filter>
+                <categories>
+                  <li>Textiles</li>
```

Comment:
This would include all textiles, even cloth and wool that ultimately the new beddings can't be made of. Once more in the name of clarity, it would be swell if we could limit the filter to a "leather" category. But I'm not sure if it exists. Gotta check it up.

Reply:
I'm confused by this one, I just copied it from the other patches.

Would the following work? My gut says no because I think leathery works like metallic, woody, stony.

<li>Leathery</li>

Reply:
Yeah, Rimword categories for things are very confusing. There are categories, stuffCategories, thingCategories.... leave it like that, I'll figure it out later.